Running `horizon db backfill` without arguments panics

```
panic: runtime error: index out of range

goroutine 1 [running]:
main.glob..func1(0x13b1600, 0x1414450, 0x0, 0x0)
	/home/travis/gopath/src/github.com/stellar/go/services/horizon/db.go:35 +0x18d
github.com/stellar/go/vendor/github.com/spf13/cobra.(*Command).execute(0x13b1600, 0x1414450, 0x0, 0x0, 0x13b1600, 0x1414450)
	/home/travis/gopath/src/github.com/stellar/go/vendor/github.com/spf13/cobra/command.go:603 +0x241
github.com/stellar/go/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc0000e5440, 0xc000028118, 0x0, 0xc000133f70)
	/home/travis/gopath/src/github.com/stellar/go/vendor/github.com/spf13/cobra/command.go:689 +0x2cc
github.com/stellar/go/vendor/github.com/spf13/cobra.(*Command).Execute(0xc0000e5440, 0xc000133f88, 0xc000028118)
	/home/travis/gopath/src/github.com/stellar/go/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/home/travis/gopath/src/github.com/stellar/go/services/horizon/main.go:23 +0x2d
```

--- a/services/horizon/db.go
+++ b/services/horizon/db.go
@@ -25,6 +25,11 @@ var dbBackfillCmd = &cobra.Command{
 	Use:   "backfill [COUNT]",
 	Short: "backfills horizon history for COUNT ledgers",
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			log.Println("Missing COUNT. Usage: backfill [COUNT].")
+			return
+		}
+
 		app := initApp(cmd, args)
 		app.UpdateLedgerState()
 

